continuous-delivery/musl-linux-build.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
This file doesn't have anything musl specific. Should we rename it to generic-unix-build.sh?